### PR TITLE
Add polling place count

### PR DESCRIPTION
**Pull request by tank157**
Adds polling_place_count to the results file.

I also added some debug logging we can turn on in dev to see the progress and debug issues. And I added a queue declare for the `batch-address.file.submit` queue so we can start up without needing to have it created first by Metis.

[Pivotal Card](https://www.pivotaltracker.com/story/show/150650489)

### CODE REVIEW

File: src/vip/batch_address_test_tool/processor.clj [hunk 1]
```diff
@@ -147,12 +155,15 @@
...
   "Publishes response message to output queue"
   [ctx]
-  (if (contains? ctx :error)
-    (q/publish-to-queue {"status" "error"
-                         "error" {"message" (.getMessage (:error ctx))}
-                         "groupName" (get-in ctx [:input "groupName"])
-                         "transactionId" (get-in ctx [:input "transactionId"])}
-                        "batch-address.file.complete")
+  (log/debug "respond:" (pr-str ctx))
+  (if-let [error (:error ctx)]
```
Comment (katrinamariehh) [inline src/vip/batch_address_test_tool/processor.clj:159]: This all looks good--can you explain how the `if-let` works in the context of the rest of this function?

Comment (tank157) [reply src/vip/batch_address_test_tool/processor.clj:159]: So before the `respond` code was looking for an error and just publishing an error response back. This works, but doesn't give any good log info when an error occurs. So the `if-let` just allows us to bind the error value (when it exists), and then pop into the `do` and both log an error and send the response. If no error existed, we fall in to the `else` branch of the `if-let` and do a standard, good response.

**Comment by katrinamariehh** [comment]
Thanks for the additional explanation!  This looks good to me -- have we deployed the code to staging and run some tests?

**Comment by tank157** [comment]
Not yet, that could be a good next step though, now that the PR has been approved.

**Pull request merged**